backend/app/api/v1/lantern_api.py

```
@@ -17,6 +21,55 @@
 
 
+@router.get(
+    "/lanterns/{lantern_id}/music-status",
+    responses={
+        200: success_200_music_status,
+    }
+)
+async def music_status(
+    request: Request,
+    lantern_id: str = Path(...),
+    db=Depends(get_mongo_client)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)